webserver/errors.py

```diff
@@ -1,6 +1,41 @@
 from flask import render_template, make_response
+from yattag import Doc
+import yattag
+import ujson
 
 
+# List of errors compatible with LastFM messages for API_compat.
+ERROR_MESSAGES = {
+    1: "This error does not exist",
```

Comment:
This is a good way of storing the error messages, but I'm a little worried about using the numbers in the rest of the views. This is what's called a "magic number" (https://en.wikipedia.org/wiki/Magic_number_(programming)), and is generally considered a bad idea, because it makes the context in which the number is being used unclear.
Unfortunately, python 2.7 doesn't have an enumerator type, which makes this kind of pattern really annoying to do. 
This solution is a pretty reasonable idea: http://stackoverflow.com/a/36937
We could have a
```python
class CompatError:
    DOES_NOT_EXIST = 1
    INVALID_SERVICE = 2
   ... etc

ERROR_MESSAGES = {DOES_NOT_EXIST: "This error does not exist", ...}
```
You can make up names for each of the variables, just keep them short but understandable and unique among all errors.